src/main/java/com/minecolonies/coremod/colony/managers/StatisticsManager.java

```diff
@@ -51,14 +52,14 @@ public void increment(final @NotNull String id)
     @Override
     public void incrementBy(final @NotNull String id, int qty)
     {
-        final Map<Short, Integer> innerMap = stats.computeIfAbsent(id, k -> new HashMap<>());
+        final Map<Short, Integer> innerMap = stats.computeIfAbsent(id, k -> new Short2IntOpenHashMap());
```

Comment:
You don't get most of the performance benefits unless you access it as its own class and not the base `Map<>` class.  Otherwise you're still (un)boxing.